src/app/app.component.ts

```diff
@@ -28,6 +31,11 @@ export class AppComponent implements AfterViewInit, OnInit {
     }
   ]).sort();
 
+  @ViewChild(AlertComponent, {static: false})
+  alert: AlertComponent | undefined;
+
+  alertSubject$: BehaviorSubject<boolean> = new BehaviorSubject(false);
```

Comment:
Behavior subjects are observables, and the standard approach when it comes to naming them is to prefix the variable name with the `$` sign. 

Here is an example:

```diff
- alertSubject$: BehaviorSubject<boolean> = new BehaviorSubject(false);
+$alert: BehaviorSubject<boolean> = new BehaviorSubject(false);
```

Reply:
According to the documentation, using dollar signs as a suffix rather than a prefix adheres to common conventions, cf. <https://angular.io/guide/rx-library#naming-conventions-for-observables>